src/test/java/core/basesyntax/StorageTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class StorageTest {
+    private static final String APPLE = "apple";
```

Comment:
```suggestion
    public static final String APPLE = "apple";
```

Reply:
https://prnt.sc/xzEg0vfGGXa0

Reply:
https://prnt.sc/xzEg0vfGGXa0 - screen of my test coverage of tests